packages/react-admin/src/mui/layout/DashboardMenuItem.js

```diff
@@ -16,20 +16,22 @@ const styles = {
     },
 };
 
-const DashboardMenuItem = ({ classes, onClick, translate }) => (
+const DashboardMenuItem = ({ classes, onClick, translate, props }) => (
```

Comment:
Not what I meant. Why don't you simply pass all extra props ?

```js
const DashboardMenuItem = ({ classes, onClick, translate, ...props }) => (
```

Reply:
I was following the pattern used in most input components, e.g. `BooleanInput`, `TextInput`. Should I still change it?

Reply:
Yes please, we're going to apply this to all components anyway for `next`

Reply:
Besides, sorry but you'll have to rebase as we updated `next` to add `className` to all components too